packages/focal-atom/test/atom.test.ts

```diff
@@ -744,7 +744,7 @@ describe('atom', () => {
   describe('fromObservable', () => {
     test('emits atom', async () => {
       const a = await Atom.fromObservable(from([1])).pipe(take(1)).toPromise()
-      expect(a.get()).toEqual(1)
+      expect(a?.get()).toEqual(1)
```

Comment:
`rxjs@7` https://rxjs.dev/6-to-7-change-summary#observable
> `toPromise` method now correctly returns `Promise<T | undefined>` instead of `Promise<T>`. This is a correction without a runtime change, because if the observable does not emit a value before completion, the promise will resolve with `undefined`.